qml/Tutorial/TutorialContent.qml

```diff
@@ -77,15 +77,15 @@ Item {
 
         readonly property bool skipped: !d.mobileScenario || d.haveShown("top")
         readonly property bool shown: item && item.shown
-        active: !skipped || (item && item.visible)
+        active: !skipped
```

Comment:
What does this fix?

Reply:
Some `TypeErrors`

Reply:
Would this not mean if Tutorial still ran even if its "skipped" it would make it inactive? I assume the or there was to keep it active when its shown regardless if skipped is true.

Reply:
In what cased would it ever be skipped and still visible? AFAICT, it should only ever be visible if it's not skipped and wasn't already completed. I don't see why it would ever be skipped and still visible.